### ISSUE TITLE
Inclusivity for non-native english speakers

### ISSUE THREAD

**Issue by juliepagano** [nodejs/inclusivity#20]
Creating an issue for discussion of this topic since it came up a bunch in several other issues. There are several non-native english speakers participating in the WG, so hopefully they can provide useful input.

Some specific things that came up. Let me know if I poorly summarized.
- Documentation should be written in a way that is understandable to non-native English speakers.
- Language confusion is easy, particularly when someone is not a native english speaker. Consideration should be made for assuming good intent when issues arise around language confusion.

**Comment by sup** [comment]
as a non-native english speaker, i think the points you listed are a good start. maybe another nice thing would be easier access to localization resources, how to contribute to localization and, once they're finished, how to easily access the resources provided by the respective localization team.

**Comment by yosuke-furukawa** [comment]
That would be super helpful. Straightforward and easy docs are translated easily.

**Comment by thefourtheye** [comment]
When we talk about documentation, I believe most developers would like to see working examples and straight to the point description. So we will always have room for improvement.

Now, if somebody doesn't understand the language used in the docs, how will they get help with that? For example, I myself had a similar experience with the expressjs docs and ended up [posting a question in Stackoverflow](http://stackoverflow.com/q/20550656/1903116) and [creating a bug](https://github.com/strongloop/express/issues/1859). But, I feel that, most of the times, creating a bug in the main repo for assistance may not be a good idea.

Basically my concern boils down to, how people will reach out to us if they need help with understanding the docs text or pointing out mistakes in the docs.

**Comment by javascriptdev2000** [comment]
What I am concerned about is whether non-native English speakers could easily understand whether a word has problematic connotations.

I would suggest creating a listing of such terms, for example:
* Master/slave/servant
* Blacklist
* suicide-related terminology
* Eggplants
* :eggplant: 🍆🍆🍆🍆🍆🍆🍆🍆🍆🍆  :eggplant:

**Comment by TronaldDump** [comment]
I have spoken to a number of African-American developers regarding the lack of diversity in tech, and a number of them raised the important question of, I quote, "where mah niggaz at". I am unsure as to the meaning of this phrase (I would be grateful if one of you could explain it!) but it appears that the lack of knowledge regarding "where mah niggaz at" is a major barrier to African-Americans who may wish to contribute to this and other projects.

As part of our outreach to the African-American community, I suggest that we include a prominent, clear and concise explanation of "where mah niggaz at", preferably in African-American Vernacular English. In addition, it would perhaps aid the comprehension of our African-American brethren* if our documentation included various other phrases from their vocabulary, such as "bix nood", "he a good boy" and "dindu nuffin", reference to shooting cops, etc.

It would be SUPER great if someone could get on this right away. Thanks! :+1: 

 *Is this a problematic word? If so, I apologize.

**Comment by 950c** [comment]
As a non-native English speaker I have had absolutely no problem with reading your documentation, and I think it is completely OK to assume that people will have a reasonable understanding of English before they try to read English documentation.

The ones that don't want to bother (seen this with Japanese, German etc) will just get somebody to translate for them.

**Comment by sup** [comment]
@950c as though i partially agree with you, i think there are some phrases that are harder to understand for people with lesser knowledge of english. we should at least try and rephrase those

**Comment by ravi** [comment]
A small point: it may also be worth keeping in mind that there are many versions of "native" English (British, American, Canadian, Australian, Indian, so on), and accompanying idioms, norms, so on.

**Comment by Wysaard** [comment]
As a non-native English speaker (Dutch) I think proper English should be the norm, and language should never be dumbed down. Documentation should be written in the clearest way possible, even if that means using a 'hard word' sometimes.

Just as a reminder: There is such a thing as an online translator.

Also, English in CS is what Latin used to be for the sciences: Everyone who's seriously involved knows it, and those who want to get involved should learn it. It's as simple as that.

If anything, we should have a standard of using terms that aren't really exclusive to any of the versions of native english, and set the standard to be either British or American. Of course, without throwing a fit whenever someone doesn't follow this standard.

**Comment by Fishrock123** [comment]
> A small point: it may also be worth keeping in mind that there are many versions of "native" English (British, American, Canadian, Australian, Indian, so on), and accompanying idioms, norms, so on.

Node Core uses and aims to use US English. I forget whether or not it's officially stated anywhere, and I don't really know the historical context other than Node.js originated in the US.

> - Documentation should be written in a way that is understandable to non-native English speakers.

I think this is a good idea, it comes down to clarity, really. I'd bet it mostly means avoiding cultural [colloquialisms](https://en.wikipedia.org/wiki/Colloquialism) other than that.

**Comment by scottgonzalez** [comment]
The title of this issue is fairly vague and the original post mentions "There are several non-native english speakers participating in the WG" but the comments all seem to be around documentation. I'd like to point out that the first WG meeting was a video call. I know this came up in the meeting, and I've personally heard requests for text-based meetings in the past (for other projects that I manage): People who aren't fluent in English have a hard time keeping up with the conversation on calls. It's easier to keep track of, and participate in, the conversation if the meeting is written out in text. Unfortunately, the benefits of actually talking are fairly high for those who do speak English fluently. The W3C WGs that I've participated in do both: They have a voice call but also have a scribe who writes notes in real time in IRC. Perhaps this would be something we can do for the inclusivity meetings as well.

**Comment by TheAlphaNerd** [comment]
This would also serve to benefit those with hearing loss.

**Comment by zkat** [comment]
@scottgonzalez I really like that idea -- do you mind creating a separate issue for it as an agenda-item for a future meeting?

**Comment by scottgonzalez** [comment]
> @scottgonzalez I really like that idea -- do you mind creating a separate issue for it as an agenda-item for a future meeting?

Done: #54.

**Comment by ravi** [comment]
>> A small point: it may also be worth keeping in mind that there are many versions of "native" English (British, American, Canadian, Australian, Indian, so on), and accompanying idioms, norms, so on.
>
> Node Core uses and aims to use US English. I forget whether or not it's officially stated anywhere, and I don't really know the historical context other than Node.js originated in the US.

I think that is quite fine, but my point, if minor, was a bit different, and related to the second of @juliepagano's two bullet points above. To wit, it is not just non-native speakers that should be taken into consideration, but also, the idea of "native" being poorly defined, it may be worthwhile to consider that different native English speakers have different traditions for the same language. For instance, an English speaker from India or Zimbabwe might not understand an idiom that is indigenous to, say, the midwest USA. And vice versa (of course we can, as a blanket rule, eschew idioms as obfuscatory, but I suspect it will be impossible to eliminate every such artefact, and hence, perhaps, the attention to inclusivity?).

FWIW.
